examples/dbclient/jdbc/src/main/java/io/helidon/examples/dbclient/jdbc/JdbcExampleMain.java

```diff
@@ -96,8 +96,13 @@ private static Routing createRouting(Config config) {
         DbClient dbClient = DbClient.builder(dbConfig)
                 .build();
 
+        // Some relational databases do not support DML statement as ping so using query which works for all of them
         HealthSupport health = HealthSupport.builder()
-                .addLiveness(DbClientHealthCheck.create(dbClient))
+                .addLiveness(
```

Comment:
Is the `query` required? It would be good to have a configuration based approach that does not require any additional code.
When I change between databases (and one supports Query and the other DML statement for health) I do not want to change code.
It would be good to have `DbClientHealthCheck.create(dbConfig)` or similar, to be able to define whether it is a `query`  or `dml` without coding.

Reply:
Default is DML, but not all databases support DML statement as ping. Query works for everything including Mongo.
I can solve it by changing default to query.

Reply:
Pushed modified DbClientHealthCheck API. Will modify this example to use setup from config.

Reply:
Examples updated and tested that health check still works.